libcontainer/cgroups/systemd/v2.go

```diff
@@ -302,8 +301,10 @@ func (m *unifiedManager) Destroy() error {
 	}
 
 	// XXX this is probably not needed, systemd should handle it
-	err := os.Remove(m.path)
-	if err != nil && !os.IsNotExist(err) {
+	logrus.Warnf("try to destroy cgroup: %s", m.path)
```

Comment:
Why is this a warning? Why log here at all?

Reply:
Sorry for commit mistake. I changed it to Debugf